recipes/yews/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "yews" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "0f5ef8c0fcf5892ebe7c17205f11d22e8b6db01c1ce7df45850d5337f98975db" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}  
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:  
+  host:
+    - pip
+    - python>=3.6
```

You need a space here for some of the conda-forge scripts to work:
```suggestion
    - python >=3.6
```
(and for the other two)

---

Thanks a lot for the suggestion. I did not know that before.

---

The other two version constraints need fixing. (IIRC this causes feedstock creation to fail)

---

@chrisburr I thought that rule only applies to `host`. They have been fixed now.